src/Input.test.jsx

```diff
@@ -0,0 +1,21 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import Input from './Input';
+
+test('Input 컴포넌트에 props로 전달된 이벤트 핸들러가 정상적으로 호출된다.', () => {
+  const handleClick = jest.fn();
+  const handleChange = jest.fn();
+
+  const { getByText, getByPlaceholderText } = render(
+    <Input onChange={handleChange} onClick={handleClick} value="" />,
```

Comment:
```suggestion
    <Input
      onChange={handleChange}
      onClick={handleClick}
      value=""
    />
```

저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
코드 가독성을 고려한 자기만의 코드 정리 기준을 세우시면 좋아요.